src/plugins/selecttext_onfocus_and_autoselectoption_on_typed/plugin.js

```diff
@@ -0,0 +1,59 @@
+Selectize.define('selecttext_onfocus_and_autoselectoption_on_typed', function(options) {
```

Comment:
Would this make more sense as two separate plugins?

Reply:
Done.